frontend/src/lib/i18n/locales/en.json

```diff
@@ -155,7 +155,9 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
       "modal_title": "Add a Member to this project",
       "submit_button": "Add Member",
       "user_not_found": "No user was found with this email address",
-      "user_not_verified": "User has not verified their email address"
+      "user_not_verified": "User has not verified their email address",
+      "modal_message": "The message you added in the modal",
```

Comment:
What's this for?

Reply:
😆 I removed that locally, but forgot to push.
That's what cursor.sh generated for me. 🎉 😝